app/views/layouts/_footer.html.erb

```diff
@@ -23,21 +23,21 @@
       <div class="col-xs-12 col-sm-12 col-md-4 col-lg-4 footer-column">
         <div class="row text-left">
           <div class="col-5 footer-links">
-            <a href="/simulator"><h6><%= t("layout.link_to_simulator") %></h6></a>
-            <a href="/learn" target="_blank"><h6><%= t("layout.link_to_learn_more") %></h6></a>
-            <a href="https://blog.circuitverse.org" target="_blank"><h6><%= t("layout.link_to_blog") %></h6></a>
-            <a href="/examples"><h6><%= t("layout.footer.link_to_examples") %></h6></a>
+            <h6><%= t("layout.link_to_simulator") %><a href="/simulator"></a></h6>
```

Comment:
It seems these changes have rendered the text unclickable. Could you kindly verify @yogeshsshekhawat.

Reply:
Bhaiya before the changes the full block was click able  and after the changes only text is click able 

You can also refer to error #4409

Reply:
@yogeshsshekhawat the links can be unclickable.
Instead modify the CSS to limit the width of each a tag.

You can set this properties to make the width accoriding to text, which will resolve the issues
```
  width: fit-content;
  display: block;
```